### DISCUSSION TITLE
No Rows Found

### DISCUSSION THREAD

**Discussion by syeillah** [Budibase/budibase#4994] [Help]
Hi, want to ask something

So im using postgreSQL table, usually all the data entered in the form will be listed here, but this time the data entered does not enter this table and causes an error

<img width="1395" alt="Screen Shot 2022-03-22 at 09 14 15" src="https://user-images.githubusercontent.com/99949174/159394710-69209f48-9fd4-48c4-a451-2dca3c0e50c1.png">

May i know the possible reason? is it bcs the network or another factors?

**Comment by melohagan** [comment]
Hi @syeillah 

Just to make sure, double check there definitely are rows in the PostgreSQL table, and that you are connected to the correct database and schema. 

Then open the network tab of the dev-tools (F12 on Chrome). 
Click any other datasource, and then click back on 'FTS Plan Visit' and report back any error codes / messages you see in the network tab.
